lib/cylc/scheduler.py

```diff
@@ -580,6 +580,7 @@ def _task_type_exists(self, name_or_id):
             name = TaskID.split(name_or_id)[0]
         return name in self.config.get_task_name_list()
 
+    @staticmethod
     def get_standardised_point_string(self, point_string):
```

Comment:
Need to remove `self` from argument list of `@staticmethod`. Tests should pass when this is done.